pkg/stub/handler.go

```diff
@@ -487,7 +477,15 @@ func (h *Handler) CreateDefaultResourceIfNeeded(cfg *v1.Config) (*v1.Config, err
 func (h *Handler) initConditions(cfg *v1.Config) *v1.Config {
 	now := kapis.Now()
 	util.Condition(cfg, v1.SamplesExist)
-	util.Condition(cfg, v1.ImportCredentialsExist)
+	creds := util.Condition(cfg, v1.ImportCredentialsExist)
```

Comment:
not deleting this condition in case customers are coding to it

could entertain marking deprecated in openshift/api